box.go

```diff
@@ -150,6 +150,29 @@ func genkey(publicKeyFile string, privateKeyFile string) {
 	pemWrite(privateKey, privateKeyFile, "NACL PRIVATE KEY", 0600)
 }
 
+func decryptEnvelopes(input string, decryptor DecryptionStrategy) (output string, err error) {
```

Comment:
Please add test case where the base64 encoded envelope contains whitespaces and newlines, for example if it's broken into fixed-width blocks. I think there may be a stripWhitespace lacking somewhere in this function, like the decryptStream() and decryptEnvironment() commands do